playbooks/install_stack.yaml

```diff
@@ -380,6 +380,33 @@
         cip_content:
           parameter_defaults:
             ContainerImagePrepare: "{{ cip_config }}"
+            ContainerImageRegistryLogin: true
+
+    # Red Hat Service Accounts have fixed usernames that incorporate a pipe character
```

Comment:
I'm not sure I understand why we need that now.

Reply:
regarding the ContainerImageRegistryLogin: true in the cip config, this is again because the rhcs5 piece is handled differently than the rest of the cloud install

this is again only for rhsm based installs